bundles/AdminBundle/Resources/public/js/pimcore/helpers/generic-grid.js

```diff
@@ -46,17 +47,13 @@ pimcore.helpers.grid.buildDefaultStore = function (url, fields, itemsPerPage, cu
             read: 'POST',
             update: 'POST',
             destroy: 'POST'
-        }/*,
+        },
         listeners: {
             exception: function(proxy, response, operation){
-                Ext.MessageBox.show({
```

Comment:
Since this is a generic grid, which might be used in multiple places, is it really a good idea to change the messagebox to a notification?

Reply:
Yes, I know. In my opinion it is because otherwise the user will just see nothing in case of errors. But if you are afraid we can duplicate this code and use the notification window only for predefined metadata.

Reply:
@BlackbitDevs I am also confused about this one. Maybe @dvesh3 has an opinion about it?